eng/scripts/automation/GenerateAndBuildLib.ps1

```diff
@@ -73,19 +79,56 @@ function Update-AutorestConfigFile() {
                     $configline = "input-file:"  + "$inputfile"
                 }
             }
-            $fileContent = Get-Content -Path $autorestFilePath
+
             $fileContent[$startNum - 1] += ([Environment]::NewLine + $configline)
             $fileContent | Set-Content $autorestFilePath
             if ( !$? ) {
                 Write-Error "Failed to update autorest.md. exit code: $?"
                 exit 1
             }
-        }  
+        }
+        
+        # update autorest.md with configuration
+        if ( $autorestConfigYaml -ne "") {
```

Comment:
Tip: just `if ($autorestConfigYaml)` is fine. If null or empty, it will evaluate as false anyway.